gradle.properties

```diff
@@ -21,4 +21,6 @@ org.gradle.jvmargs=\
   --add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED \
   --add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED \
   --add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED \
-  --add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED
+  --add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED \
+  -DdefaultPostgresVersion=14
```

Comment:
Adding one more property when we already have a property `it.nessie.container.postgres.tag`, doesn't make sense to me. 

I think the problem is that if we need to update the postgres container version, we need to upgrade in two places. We need to avoid this by updating in one place.

I see two approaches

a. Declare a variable in gradle (`pgContainerVersion=14`) and use it in java code.
Something like this (https://stackoverflow.com/questions/17197636/is-it-possible-to-declare-a-variable-in-gradle-usable-in-java)
I am also new to gradle, I need to do this by trial and error or googling.

b. Another approach is to have a Util class with the static method in the common shared test code.
I tried it and it worked (https://github.com/projectnessie/nessie/commit/0c93c75afed430cfc97c86bc563a9c97e5821939)

I would still prefer option a)

Also please rebase your code (work on the latest code), this PR seems to have conflicts.